### ISSUE TITLE
About loc model

### ISSUE THREAD

**Issue by mingjiayang** [cy94/sceneformer#3]
Thanks for your great paper and code.

I used the configuration provided in `configs/scene_shift_X_config.yaml` to train 4 models.
The minimum losses of the `cat, dim, ori` models are about 1.5, and the corresponding epochs are about 300. But the loss of `loc` model can't be minimized to less than 2.5, and the training stopped early at about 100 epochs. 

Does the training of the `cat, dim, ori` models meet expectations?
And what configuration should I modify to fix the training of model `loc`? (e.g. for `configs/scene_shift_loc_config.yaml`, dose `warmup: 2` need to modified?)

**Comment by cy94** [comment]
Thanks for your interest. A higher loss with the `loc` model is expected as the number of possible outputs is higher, and there is more variation. You could try a lower learning rate and higher warmup (`2` seems to be a typo since thats the number of iterations) and then make further changes based on the loss curves, such as increasing the model capacity.

**Comment by mingjiayang** [comment]
Thanks, I will try it.

**Comment by lbf4616** [comment]
Hi, 兄弟，你是用的SUNCG数据集吗？能分享下吗？

**Comment by JackW987** [comment]
> Thanks, I will try it.

Excuse me,I get some problem after I run the deep-synth data prepare scripts. When I start training , I get an error named "No such file named model_dims.pkl",but I just have 7000+ images and four files named "coarse_categories_frequency" ,"final_categories_frequency","fine_categories_frequency","model_frequency","model_prior.pkl", Is my data prepared running wrong?

**Comment by mingjiayang** [comment]
@JackW987 You can get `model_dims.pkl` by running [obj_dims.py](https://github.com/brownvc/fast-synth/blob/master/scene-synth/obj_dims.py) in `fast-synth`.
